packages/vm/src/evm/opcodes/util.ts

```diff
@@ -258,6 +263,50 @@ export function updateSstoreGas(
       -> Value is zero, but slot is nonzero
       Thus, the remaining case is where value is nonzero, but slot is zero, which is this clause
     */
-    return new BN(common.param('gasPrices', 'sstoreSet'))
+    return BigInt(common.param('gasPrices', 'sstoreSet'))
+  }
+}
+
+export function mod(a: bigint, b: bigint) {
+  let r = a % b
+  if (r < 0n) {
+    r = b + r
+  }
+  return r
+}
+
+export function fromTwos(a: bigint) {
+  return BigInt.asIntN(256, a)
+}
+
+export function toTwos(a: bigint) {
+  return BigInt.asUintN(256, a)
+}
+
+export function abs(a: bigint) {
+  if (a > 0) {
+    return a
+  }
+  return a * -1n
+}
+
+export function bufferToBigInt(buf: Buffer) {
```

Comment:
would this be better placed in ethereumjs-util?

Reply:
I'm not opposed to it.  I was just trying to limit the amount of change outside the VM package when I wrote that.

Reply:
i suggest we add a few functions to ethereumjs-util that you do repeatedly across this PR. then let's use them in all the code, so for example if we find a faster way to do the function, then we automatically get the updated benefits across all packages.

to start, I am thinking `bufferToBigInt` and `bigIntToBuffer`

I would suggest also adding `bnToBigInt` and `bigIntToBN` but if we plan to remove bn.js completely then they would also become unsupported. For now bn -> BigInt can just do in one style so it's easier to search/replace later, maybe stick to `BigInt(bn.toString(10))` for consistency.

Reply:
For the `bigIntToBuffer` route, does it make more sense to just add to `toBuffer`?